setup.py

```diff
@@ -51,8 +51,8 @@
     },
 
     install_requires=[
-        'pylint==1.4.5',
-        'pylint-django==0.6.1',
+        'pylint>=1.5.4,<2.0.0',
```

Comment:
When I saw this pull request, I thought, "wait, didn't we do 3.5 already? Oh yeah, there was something that made it difficult."  This is the thing.  We could pin the version of pylint in edx-platform, but there's a larger problem: it's natural for pylint to add new checkers in new versions.  This means that when pylint upgrades, there will likely be more violations found.  Builds that depend on not exceeding some threshold of violations will then break. We don't want pin-ranges to mean that we could silently upgrade something that will break our builds.  I'm not sure what to do about this.